UPGRADE-6.4.md

```diff
@@ -89,8 +89,11 @@ HttpFoundation
 HttpKernel
 ----------
 
+ * Deprecate `UriSigner`, use `HttpFoundation` `UriSigner` instead
  * [BC break] `BundleInterface` no longer extends `ContainerAwareInterface`
  * [BC break] Add native return types to `TraceableEventDispatcher` and to `MergeExtensionConfigurationPass`
+ * `BundleInterface` no longer extends `ContainerAwareInterface`
+ * Add native return types to `TraceableEventDispatcher` and to `MergeExtensionConfigurationPass`
```

Comment:
Hmm, these 2 seems unrelated. I think maybe something went wrong resolving the merge conflict?